imports/ui/QuestionSidebar.jsx

```diff
@@ -29,7 +29,7 @@ export class QuestionSidebar extends ControlledForm {
     super(props)
     this.state = { questionPool: this.props.questions.slice(),
       questionType: -1,
-      courseId: -1,
+      courseId: this.props.courseId,
```

Comment:
Is it redundant to have courseID as state and prop? Feels wrong, if avoidable, since we would could end up in a situation where they could be different.